test/e2e/agent/plugins/common/src/test/java/org/apache/shardingsphere/test/e2e/agent/common/env/E2ETestEnvironment.java

```diff
@@ -72,81 +98,93 @@ public static E2ETestEnvironment getInstance() {
     }
     
     /**
-     * Prepare environment.
+     * Start.
```

Comment:
this is not a good comment.

Reply:
Method modified to init(), and comment modified Init environment